home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1171,6 +1173,57 @@ paths:
         default:
           $ref: '#/components/responses/InternalServerError'
 
+  /polls:
+    get:
+      tags:
+        - poll
+      summary: Get all polls
+      description: The endpoint gets all polls
+      operationId: queryPoll
+      parameters:
+        - $ref: '#/components/parameters/page_number'
+        - $ref: '#/components/parameters/page_size'
+        - $ref: '#/components/parameters/sort'
+        - $ref: '#/components/parameters/filter'
+        - $ref: '#/components/parameters/cooperation_id'
```

Comment:
Is that parameter mandatory here?

Reply:
Let me put the idea of why I'm asking it here. All the resources should be cooperation scoped. So we need to be sure that we are following it.

Reply:
Ok, I'll make it mandatory.